python/ray/projects/scripts.py

```diff
@@ -110,3 +117,130 @@ def create(project_name, cluster_yaml, requirements):
     "session", help="[Experimental] Commands working with ray session")
```

Comment:
I know this is not part of this PR, but this should have a bit more explanation. Most people won't know what a "ray session" is. How about we say something like "Commands for working with sessions, which are running instances of a project"? I'll also try clarify in the docs, but people shouldn't have to look at the docs if they don't need to.